### PR TITLE
Update to use new logging, Lykke.Sdk and fix broadcast method

### PR DESCRIPTION
**Pull request by andreykrivoshapov**

### CODE REVIEW

File: src/Lykke.Service.Dash.Api.Core/Domain/Health/HealthIssue.cs [hunk 1]
```diff
@@ -1,17 +1,17 @@
-﻿namespace Lykke.Service.Dash.Api.Core.Domain.Health
-{
-    public class HealthIssue
-    {
-        public string Type { get; private set; }
-        public string Value { get; private set; }
+﻿////namespace Lykke.Service.Dash.Api.Core.Domain.Health
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api.Core/Domain/Health/HealthIssue.cs]: Remove commented code

File: src/Lykke.Service.Dash.Api.Core/Domain/Health/HealthIssuesCollection.cs [hunk 2]
```diff
@@ -1,32 +1,32 @@
-using System.Collections;
-using System.Collections.Generic;
+﻿////using System.Collections;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api.Core/Domain/Health/HealthIssuesCollection.cs]: Remove commented code

File: src/Lykke.Service.Dash.Api.Core/Services/IHealthService.cs [hunk 3]
```diff
@@ -1,12 +1,12 @@
-﻿using System.Collections.Generic;
-using Lykke.Service.Dash.Api.Core.Domain.Health;
+﻿//using System.Collections.Generic;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api.Core/Services/IHealthService.cs]: Remove commented code

File: src/Lykke.Service.Dash.Api.Core/Services/IShutdownManager.cs [hunk 4]
```diff
@@ -1,9 +1,9 @@
-﻿using System.Threading.Tasks;
+﻿//using System.Threading.Tasks;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api.Core/Services/IShutdownManager.cs]: One more time

File: src/Lykke.Service.Dash.Api.Core/Services/IStartupManager.cs [hunk 5]
```diff
@@ -1,9 +1,9 @@
-﻿using System.Threading.Tasks;
+﻿//using System.Threading.Tasks;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api.Core/Services/IStartupManager.cs]: And again

File: src/Lykke.Service.Dash.Api/Services/DashService.cs [hunk 6]
```diff
@@ -113,27 +113,22 @@ public Transaction GetTransaction(string transactionHex)
...
-                    throw new ArgumentException($"{nameof(response)}{nameof(response.Txid)} can not be null or empty. Response={response}");
-                }
             }
             catch (Exception ex)
             {
-                await _log.WriteErrorAsync(nameof(DashService), nameof(BroadcastAsync),
-                    $"transaction: {transaction}, operationId: {operationId}", ex);
-
-                throw;
+                if (ex.ToString().Contains("transaction already in block chain"))
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api/Services/DashService.cs:124]: Looks brittle. There is no more specific exception with something kinda error code?

File: src/Lykke.Service.Dash.Api/Startup.cs [hunk 7]
```diff
@@ -1,211 +1,35 @@
...
-                    options.DescribeAllEnumsAsStrings();
-                    options.DescribeStringEnumsInCamelCase();
-                });
-
-                var builder = new ContainerBuilder();
-                var appSettings = Configuration.LoadSettings<AppSettings>();
-                Log = CreateLogWithSlack(services, appSettings);
+                    logs.AzureTableName = "DashApiLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.DashApiService.Db.LogsConnString;
+                };
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api/Startup.cs]: Add BlockChainIntegration and BlockChainIntegrationImportantMessages slack channels with min log level = Information and Warning respectively.

File: src/Lykke.Service.Dash.Job/PeriodicalHandlers/BalanceHandler.cs [hunk 8]
```diff
@@ -26,8 +30,14 @@ public class BalanceHandler : TimerPeriod
             }
             catch (Exception ex)
             {
-                await _log.WriteErrorAsync(nameof(BalanceHandler), nameof(Execute), 
-                    "Failed to update balances", ex);
+                if (_disableErrorsSending == true)
+                {
+                    _log.Debug("Failed to update balances", ex.ToString());
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Job/PeriodicalHandlers/BalanceHandler.cs]: Debug has optional ```exception``` parameter. Avoid of ```ex.ToString()```

File: src/Lykke.Service.Dash.Job/PeriodicalHandlers/BroadcastHandler.cs [hunk 9]
```diff
@@ -26,8 +29,14 @@ public class BroadcastHandler : TimerPeriod
             }
             catch (Exception ex)
             {
-                await _log.WriteErrorAsync(nameof(BroadcastHandler), nameof(Execute),
-                    "Failed to update broadcasts", ex);
+                if (_disableErrorsSending == true)
+                {
+                    _log.Debug("Failed to update broadcasts", ex.ToString());
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Job/PeriodicalHandlers/BroadcastHandler.cs]: The same thing about ```exception``` parameter

File: src/Lykke.Service.Dash.Job/Startup.cs [hunk 10]
```diff
@@ -1,224 +1,35 @@
...
+                options.ApiTitle = "Dash.Job";
+                options.Logs = logs =>
                 {
-                    app.UseDeveloperExceptionPage();
-                }
-
-                app.UseLykkeMiddleware("Dash.Job", ex => BlockchainErrorResponse.FromUnknownError(ex.ToString()), logClientErrors: true);
+                    logs.AzureTableName = "DashJobLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.DashJob.Db.LogsConnString;
+                };
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Job/Startup.cs]: The same thing about additional Slack channels

**Review by KonstantinRyazantsev — CHANGES_REQUESTED** [review]

File: src/Lykke.Service.Dash.Api/Services/DashService.cs (hunk 6 above)
Comment (andreykrivoshapov) [reply src/Lykke.Service.Dash.Api/Services/DashService.cs:124]: Yeah. Agree. Node returns 400 error and the body contains this description... It is like a quick fix for just test env. I am planning to add normal check if hash is already in blockchain or not before calling broadcast() later

File: src/Lykke.Service.Dash.Api/Startup.cs [hunk 11]
```diff
@@ -17,6 +19,18 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 {
                     logs.AzureTableName = "DashApiLog";
                     logs.AzureTableConnectionStringResolver = settings => settings.DashApiService.Db.LogsConnString;
+
+                    logs.Extended = extendedLogs =>
+                    {
+                        extendedLogs.AddAdditionalSlackChannel("BlockChainIntegration", channelOptions =>
+                        {
+                            channelOptions.MinLogLevel = Microsoft.Extensions.Logging.LogLevel.Information;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Dash.Api/Startup.cs:27]: Just FYI - ```Information``` is default value for ```MinLogLevel```, so it could be omitted.

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**